java/vector/src/main/java/org/apache/arrow/vector/BigIntVector.java

```diff
@@ -327,6 +327,14 @@ public void setWithPossibleTruncate(int index, long value) {
     this.setSafe(index, value);
   }
 
+  @Override
+  public long getValueAsLong(int index) {
+    if (NULL_CHECKING_ENABLED && isSet(index) == 0) {
```

Comment:
Sorry I think my last comment wasn't clear.  This replicates the logic in Get.  I just wanted to the logic Documented at the interface level so users know about the performance improvement.

Reply:
I see, fixed. Thanks for your pointer.